helmnet/hybridnet.py

```diff
@@ -449,13 +505,21 @@ def training_step(self, sos_batch, batch_idx):
                     )
                     terminal_logged = True
 
+        self.logger.experiment.add_scalar(
+            "val_loss",
+            loss,
+            self.trainer.global_step
+        )
+
+        self.log('val_loss', loss) # force the variable to be saved in state dict of module. tensorboard logger does not do this for some reason
```

Comment:
It probably should. Checkpointer uses this metric as a monitor, is that also wrong or no?

Reply:
I think that the checkpointer should save the model with the best validation loss, so shouldn't it be checked in the validation loop, rather than the training loop?